djvu2pdf

```diff
@@ -8,59 +8,56 @@
 initial_directory=$(pwd)
 file_in=$(readlink -f "$1")
 file_out=$(readlink -f "$2")
-file_out_basename=$(basename "$file_out")
 
 #
 # Set up the temporary storage
 # 
 
 tmpdir=$(mktemp -d)
-cd $tmpdir
+cd "$tmpdir" || exit 0
+
+echo "Using $tmpdir for temporary files."
 
 function cleanup() 
 {
-    rm -rf $tmpdir
-    cd $initial_directory
+    rm -rf "$tmpdir"
+    cd "$initial_directory" || exit 0
```

Comment:
Can we maybe remove this line entirely? I don't think that workdir changes actually persist outside of the script.